boards/arm/mimxrt1024_evk/doc/index.rst

```diff
@@ -0,0 +1,205 @@
+.. _mimxrt1024_evk:
+
+NXP MIMXRT1024-EVK
+##################
+
+Overview
+********
+
+The i.MX RT1024 expands the i.MX RT crossover processor families by providing
+high-performance feature set in low-cost LQFP packages, further simplifying
+board design and layout for customers. The i.MX RT1024 runs on the Arm®
+Cortex®-M7 core at 500 MHz.
+
+.. image:: ./mimxrt1024_evk.jpg
+   :width: 720px
+   :align: center
+   :alt: MIMXRT1024-EVK
+
+Hardware
+********
+
+- MIMXRT1024DAG5A MCU (600 MHz, 256 KB on-chip memory, 4096KB on-chip QSPI
+  flash)
+
+- Memory
+
+  - 256 Mbit SDRAM
+  - 64 Mbit QSPI Flash
+  - TF socket for SD card
+
+- Connectivity
+
+  - 10/100 Mbit/s Ethernet PHY
+  - Micro USB host and OTG connectors
+  - CAN transceivers
+  - Arduino interface
+
+- Audio
+
+  - Audio Codec
+  - 4-pole audio headphone jack
+  - Microphone
+  - External speaker connection
+
+- Power
+
+  - 5 V DC jack
+
+- Debug
+
+  - JTAG 20-pin connector
+  - OpenSDA with DAPLink
+
+For more information about the MIMXRT1024 SoC and MIMXRT1024-EVK board, see
+these references:
+
+- `i.MX RT1024 Website`_
+- `i.MX RT1024 Datasheet`_
+- `i.MX RT1024 Reference Manual`_
+- `MIMXRT1024-EVK Website`_
+- `MIMXRT1024-EVK User Guide`_
+- `MIMXRT1024-EVK Design Files`_
+
```

Comment:
Need to Hyperlink these to the documentation.

Reply:
I haven't got these links and I will going to update them when I get it.

Reply:
Are they available yet?

Reply:
According to the person in charge, it may take one to two weeks to get them.

Reply:
Ok, please update this PR when they are ready